code-review-tester-app/src/App.tsx

```diff
@@ -1,33 +1,47 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
     <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <header className="hero">
+        <h1>Code Review Tester</h1>
+        <p className="tagline">Master the art of code review in the AI era</p>
+      </header>
+      
+      <main className="content">
+        <section className="description">
+          <p>
+            In an environment where coding is increasingly done by AI models rather than humans, 
+            the ability to <strong>review code effectively</strong> has become more critical than ever. 
+            Code Review Tester is your training ground for developing sharp, efficient code review skills.
+          </p>
+          
+          <div className="features">
+            <div className="feature">
+              <h3>🎯 Choose Your Challenge</h3>
+              <p>Select your skill level and preferred programming language to get started.</p>
+            </div>
+            
+            <div className="feature">
+              <h3>📋 Review Mock Pull Requests</h3>
+              <p>Practice on realistic pull requests designed to test your review abilities.</p>
+            </div>
+            
+            <div className="feature">
+              <h3>📈 Get Expert Feedback</h3>
```

Comment:
Let's remove the word "Expert" I know you think you're an expert, but you get very confused sometimes. Let's make it clear that this is AI doing the reviewing.